server/src/config/dbconnect.js

```diff
@@ -7,6 +7,7 @@ function connect() {
     useNewUrlParser: true,
     useUnifiedTopology: true,
     useFindAndModify: false,
+    autoIndex: false
```

Comment:
Is there a specific reason for this?

Reply:
That is for optimising the creation of indices during production. [Doc Reference](https://mongoosejs.com/docs/guide.html#indexes)

Reply:
Then its better if we keep it to 
`process.env.NODE_ENV !== "production"`